python/ray/serve/controller.py

```diff
@@ -76,6 +83,386 @@ class Config:
         arbitrary_types_allowed = True
 
 
+@dataclass
+class ConfigurationStore:
+    backends: Dict[BackendTag, BackendInfo]
+    traffic_policies: Dict[EndpointTag, TrafficPolicy]
+    routes: Dict[BackendTag, Tuple[EndpointTag, Any]]
```

Comment:
```suggestion
    backends: Dict[BackendTag, BackendInfo] = field(default_factory=dict)
    traffic_policies: Dict[EndpointTag, TrafficPolicy] = field(default_factory=dict)
    routes: Dict[BackendTag, Tuple[EndpointTag, Any]]= field(default_factory=dict)
```
Python recommends using field instead of override the generated dataclass.
https://docs.python.org/3/library/dataclasses.html#default-factory-functions